shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -111,7 +111,27 @@ private Status check() {
         return Status.FAILED;
       }
 
-      return jobInfo.getStatus();
+      // This make an assumption that this job tree only goes 1 level deep
+      boolean finished = true;
+      for (JobInfo child : jobInfo.getChildren()) {
+        if (!child.getStatus().isFinished()) {
+          finished = false;
+          break;
+        }
+      }
+
+      if (finished) {
+        if (jobInfo.getStatus().equals(Status.FAILED)) {
+          System.out.println(String.format("Attempt %s to load %s failed because: %s",
+              mRetryPolicy.getAttemptCount(), mJobConfig.getFilePath(),
+              jobInfo.getErrorMessage()));
+        } else if (jobInfo.getStatus().equals(Status.COMPLETED)) {
+          System.out.println(String.format("Attempt %s to load %s completed",
```

Comment:
Perhaps word this a bit differently, `Successfully loaded path %s after %d attempts`? Also, is it possible for this to become spammy if a large number of files is to be loaded?